lib/active_record_doctor/detectors/missing_unique_indexes.rb

```diff
@@ -39,7 +39,9 @@ def detect
 
       def validations_without_indexes
         each_model(except: config(:ignore_models), existing_tables_only: true) do |model|
-          model.validators.each do |validator|
+          # Skip inherited validators from STI.
```

Comment:
Let's also explain why we're skipping: Skip inherited validators from STI to prevent them from being reported multiple times on subclasses.